### PR TITLE
Herumi master

### PR DESCRIPTION
**Pull request by herumi**
update the latest version of herumi/bls

### CODE REVIEW

**Comment by enzoh** [comment]
Thanks for the alert @timohanke, will not be able to review until tomorrow

**Review by enzoh — APPROVED** [review]
LGTM

File: ffi/go/bls/callback.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package bls
+
+/*
+// exported from bls.go
+unsigned int wrapReadRandGo(void *buf, unsigned int n);
```
Comment (eftychis) [inline ffi/go/bls/callback.go:5]: @herumi Why do we need to interface with Go?

**Comment by infinity0** [comment]
Seems to break on Darwin, can you take a look?

https://hydra.oregon.dfinity.build//build/74652/nixlog/1

~~~~
[100%] Linking CXX executable bin/bls_c256_test
Undefined symbols for architecture x86_64:
Undefined symbols for architecture x86_64:
Undefined symbols for architecture x86_64:
  "operator<<(std::__1::basic_ostream<char, std::__1::char_traits<char> >&, __mpz_struct const*)", referenced from:
  "operator<<(std::__1::basic_ostream<char, std::__1::char_traits<char> >&, __mpz_struct const*)", referenced from:
  "operator<<(std::__1::basic_ostream<char, std::__1::char_traits<char> >&, __mpz_struct const*)", referenced from:
      modTest(char const*) in bls_c384_test.cpp.o
      modTest(char const*) in bls_c256_test.cpp.o
      modTest(char const*) in bls_c384_256_test.cpp.o
ld: symbol(s) not found for architecture x86_64
ld: symbol(s) not found for architecture x86_64
ld: symbol(s) not found for architecture x86_64
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [CMakeFiles/bls_c384_test.dir/build.make:85: bin/bls_c384_test] Error 1
make[2]: *** [CMakeFiles/bls_c384_256_test.dir/build.make:85: bin/bls_c384_256_test] Error 1
make[2]: *** [CMakeFiles/bls_c256_test.dir/build.make:85: bin/bls_c256_test] Error 1
make[1]: *** [CMakeFiles/Makefile2:73: CMakeFiles/bls_c384_256_test.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:110: CMakeFiles/bls_c256_test.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:184: CMakeFiles/bls_c384_test.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
builder for '/nix/store/aq1da6cjzjk2dpanba8pn55sdqcy92mq-bls.drv' failed with exit code 2
~~~~

File: ffi/go/bls/callback.go (hunk 1 above)
Comment (herumi) [reply ffi/go/bls/callback.go:5]: DFINITY does not need the interface.
It is added for https://github.com/herumi/bls/issues/26 .

Comment (eftychis) [reply ffi/go/bls/callback.go:5]: I see 👍

**Comment by infinity0** [comment]
It might simply because clang-5.0 is too old, I've asked other devs with Mac OS X to see if we can upgrade it in our Nix definitions.

**Pull request merged**